SPECS/kubernetes/kubernetes-1.17.11.spec

```diff
@@ -18,15 +18,19 @@ URL:            https://mcr.microsoft.com/oss
 #               Note that only amd64 tarball exist which is OK since kubernetes is built from source
 Source0:        kubernetes-node-linux-amd64-%{version}-hotfix.20200901.tar.gz
 Source1:        kubelet.service
+Source2:        golang-1.15-k8s-1.17-test.patch
```

Comment:
Why not Patch4 instead of Source2? I think it's more clear

Reply:
because the file to patch (test.sh) is different in each version. test.sh changed between 1.17 and 1.18 so same .patch file cannot be applied...